README.md

```diff
@@ -221,6 +221,30 @@ module "my_db_cluster" {
 }
 ```
 
+#### Aurora Serverless
+```hcl
+module "serverless" {
+  source = "github.com/ryte/INF-tf-rds.git?ref=v0.5.1"
```

Comment:
only a minor thing, but could you format the Terraform code in the readme properly?